applications/openshift/general/kube_descheduler_lifecycle_policy/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+title: Ensure that the LifecycleAndUtilization profile for the Kube Descheduler operator is enabled
```

Comment:
I feel like this rule should be split into two,
- one rule that ensures the descheduler is run once a day: deschedulingIntervalSeconds == 86400
- one rule configuring the lifetime criteria used to deschedule: podLifetime == 24h

Reply:
I have currently no idea how to check the podLifetime. By default there is no field fpr podLifetime in the yaml, also from what I understand the value can be formatted differently. So I have to either check for the field not existing at all, for a value less than "24h" (I think the type is time.Duration) or something like "86400s"

I would rather leave this as is for now and maybe expand on this later.